tensorflow/lite/kernels/embedding_lookup_test.cc

```diff
@@ -109,6 +107,39 @@ TEST(EmbeddingLookupOpTest, SimpleTest) {
               })));
 }
 
+TEST(EmbeddingLookupOpTest, VariableShapeTest) {
+  EmbeddingLookupOpModel m({2}, {3, 2, 4});
+  m.SetInput({0, 2});
+  m.Set3DWeightMatrix(
+      [](int i, int j, int k) { return i + j / 10.0f + k / 100.0f; });
+
+  m.Invoke();
+
+  EXPECT_THAT(m.GetOutput(),
+              ElementsAreArray(ArrayFloatNear({
+                  0.00, 0.01, 0.02, 0.03, 0.10, 0.11, 0.12, 0.13,  // Row 0
+                  2.00, 2.01, 2.02, 2.03, 2.10, 2.11, 2.12, 2.13,  // Row 2
+              })));
+}
+
+TEST(EmbeddingLookupOpTest, InvalidShapeTest) {
```

Comment:
Would rename this to "IndexOutOfBoundsTest" instead of "InvalidShapeTest". The shape is technically valid ... the problem is that the input 3 is greater than max allowed value of 2. 

You can also focus on the problem we're testing by only having the input just be a single element:
e.g. m({1}, ...) and m.SetInput({3}); instead.

Reply:
@alanchiao , thanks for the input, i updated the TC as per your comments.

Regards
Amit

Reply:
The changes doesn't resolve the following point, which is applicable to all of the error cases.
 
> You can also focus on the problem we're testing by only having the input just be a single element:
e.g. m({1}, ...) and m.SetInput({3}); instead

Reply:
@alanchiao , sorry for the miss, i have updated the code as per your suggestion. Kindly check.

Regards
Amit

Reply:
Haven't quite resolved this yet. Made comments on the newest commit.